dart-impl/base/src/internal/domain_locality.c

```diff
@@ -829,15 +1009,16 @@ dart_ret_t dart__base__locality__domain__create_module_subdomains(
                    "module subdomain index:%d / num_domains:%d",
                    sd, module_domain->num_domains);
 
-    dart_domain_locality_t * subdomain = &module_domain->domains[sd];
+    module_domain->children[sd] = malloc(sizeof(dart_domain_locality_t));
+    dart_domain_locality_t * subdomain = module_domain->children[sd];
     dart__base__locality__domain__init(subdomain);
 
-    subdomain->level          = module_domain->level + 1;
-    subdomain->scope          = module_scopes[subdomain_gid_idx];
-    subdomain->relative_index = sd;
-    subdomain->global_index   = module_subdomain_gids[sd];
-    subdomain->parent         = module_domain;
-    subdomain->team           = module_domain->team;
+    subdomain->level            = module_domain->level + 1;
+    subdomain->scope            = module_scopes[subdomain_gid_idx];
+    subdomain->relative_index   = sd;
+    subdomain->global_index     = module_subdomain_gids[sd];
+    subdomain->parent           = module_domain;
+    subdomain->team             = module_domain->team;
```

Comment:
... oh yes, this flag (ignore space change) is a well-hidden commodity on github. Add the GET parameter `?w=1` or `&w=1` in the URL to disable white spaces in diffs.
Here you go:

https://github.com/dash-project/dash/pull/217/files/e770b927f9f47b1fe627d7c752e73feabcad0b49?diff=split&w=1#diff-658a755e92dabb903da18e72f17496e6